examples/adaptive-crag/application.py

```diff
@@ -161,6 +161,13 @@ def lancedb_query_template(query: str) -> str:
 </query>
 """
 
+    if chat_history:
+        chat_history_str = "\n".join(
+            [f"{msg['role']}: {msg['content']}" for msg in chat_history]
+        ).strip()
+        prompt += f"\n<chat_history>\n{chat_history_str}\n</chat_history>"
```

Comment:
Why this format? Mind a comment?

Reply:
Anthropic's models work better with xml tags like this. And I haven't experienced any degradation with other models. So I just prefer to use them.

Reply:
Got it! Might be nice adding a comment to the code about that, but yeah, that makes sense.